src/components/GrampsjsFormSelectType.js

```diff
@@ -85,9 +87,47 @@ class GrampsjsFormSelectType extends GrampsjsTranslateMixin(LitElement) {
               )}
         </mwc-select>
       </p>
+
+      <mwc-button
+        @click="${this.switchTypeInput}"
+      >
+        Enter custom type instead
```

Comment:
Please wrap this in `this._('Enter ...')` and add the string to `en.json` so it can be translated.

Would "add custom type instead" perhaps be more explicit that this will permanently create a new type, what do you think?

Reply:
I like 'Enter custom type instead' or just 'Enter custom type'. Imo 'add' refers to something being additional which is not the case, we are switching the input to custom type. I would also consider 'Switch to custom type' as we are changing the input option.

I've made these code changes and will implement with your feedback on the naming.

Reply:
Thanks, makes sense!